src/molecules/formfields/GoogleAutoCompleteField/index.js

```diff
@@ -51,6 +51,21 @@ class GoogleAutoCompleteField extends Component {
   updateAddress = () => {
     const place = this.state.autocomplete.getPlace();
     const addressData = this.formatAddressInfo(place.address_components);
+
+    if (!addressData.number) {
+      const { value } = this.props.input;
+
+      if (value) {
+        const matches = value.match(/^(\d+)/);
+
+        if (matches && matches[1]) {
+          const number = parseInt(matches[1], 10);
+
+          addressData.number = { shortName: number, longName: number };
```

Comment:
what's the difference between shortname and longname?

Reply:
oh nevermind - just realized it's a Google thing

Reply:
Yeah, I keep it in this `{ shortName, longName }` format because further processing happens later on.